spotify-app-remote/build/.transforms/a236cf5d8026cbceaedc55a536fdbf38/results.bin

```diff
@@ -0,0 +1 @@
+o/spotify-app-remote-release-0.7.2-runtime
```

Comment:
@pranavkonidena why are you committing the build folder?

Reply:
Right, my bad.. will fix this